backend/functions/lib/adapters/controllers/cases.controller.ts

```diff
@@ -1,30 +1,35 @@
 import { ApplicationContext } from '../types/basic';
 import log from '../services/logger.service';
-import { Chapter15CaseList } from '../../use-cases/chapter-15.case';
+import { CourtCaseManagement } from '../../use-cases/case-management';
 import InvalidChapterCaseList from '../../use-cases/invalid-chapter.case-list';
 
 const MODULE_NAME = 'CASES-CONTROLLER';
 
 export class CasesController {
   private readonly context: ApplicationContext;
+  private readonly courtCaseManagement: CourtCaseManagement;
 
   constructor(context: ApplicationContext) {
     this.context = context;
+    this.courtCaseManagement = new CourtCaseManagement();
   }
 
   public async getCaseList(requestQueryFilters: { caseChapter: string }) {
     log.info(this.context, MODULE_NAME, 'Getting case list.');
     if (requestQueryFilters.caseChapter == '15') {
-      const chapter15CaseList = new Chapter15CaseList();
-      return await chapter15CaseList.getChapter15CaseList(this.context);
+      return await this.courtCaseManagement.getChapter15CaseList(this.context);
```

Comment:
Should we also change this method name getChapter15CaseList ?

Reply:
As of now, the logic only fetches chapter 15 course cases, so I let be for now. We could discuss this further if we want to change or remove this.

Reply:
As stated in stand-up, I'm sympathetic to generic naming, but Mike is right in that this is highly chapter 15 specific right now. Call YAGNI for now until future refactors are more chapters are supported.